go/common/rpc/generated/enclave.proto

```diff
@@ -122,7 +122,7 @@ message SubmitTxRequest {
   bytes encryptedTx = 1;
 }
 message SubmitTxResponse {
```

Comment:
can't we use a generic message for all these methods now, since they all return the same thing?

Reply:
I was thinking of changing all of them, but I'm not sure it is a good idea from the perspective of upgradability in case we need to start adding stuff to those responses.

Reply:
if the EnclaveResponse is the new standard, then any upgrade will happen for all at once.

Reply:
I meant upgrades to the RPC layer, the enclave response is more like the enclave layer right now as from the perspective of the rpc its just bytes